app/src/main/java/com/asfoundation/wallet/ui/gamification/MyLevelFragment.kt

```diff
@@ -63,21 +66,37 @@ class MyLevelFragment : DaggerFragment(), MyLevelView {
     super.onDestroyView()
   }
 
+  override fun showNormalPioneer(pioneer: Boolean) {
+    if (pioneer) {
+      gamification_progress_bar_pioneer.setPioneer()
+      rewards_layout_pioneer.visibility = View.VISIBLE
+      rewards_layout_normal.visibility = View.GONE
+    } else {
+      rewards_layout_normal.visibility = View.GONE
+      rewards_layout_pioneer.visibility = View.VISIBLE
+    }
```

Comment:
the if is presenter logic. the view should have a method called setPioner and the presenter should have the if that calls or not that method